cmd/spanner-autoscaler/main.go

```diff
@@ -46,29 +47,28 @@ var (
 
 func init() {
 	//nolint:errcheck
-	clientgoscheme.AddToScheme(scheme)
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	//nolint:errcheck
-	spannerv1alpha1.AddToScheme(scheme)
+	utilruntime.Must(spannerv1alpha1.AddToScheme(scheme))
 
 	// +kubebuilder:scaffold:scheme
 }
 
 var (
-	metricsAddr       = flag.String("metrics-addr", "localhost:8090", "The address the metric endpoint binds to.")
-	healthzAddr       = flag.String("healthz-addr", "localhost:8091", "The address the healthz endpoint binds to.")
-	scaleDownInterval = flag.Duration("scale-down-interval", 55*time.Minute, "The scale down interval.")
+	metricsAddr          = flag.String("metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	probeAddr            = flag.String("health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	enableLeaderElection = flag.Bool("leader-elect", false, "Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	scaleDownInterval    = flag.Duration("scale-down-interval", 55*time.Minute, "The scale down interval.")
 )
 
 const (
-	exitCode             = 1
-	enableLeaderElection = true
-	leaderElectionID     = "spanner-autoscaler-leader-election"
-
-	healthzEndpoint = "/healthz"
-	readyzEndpoint  = "/readyz"
-	healthzName     = "healthz"
-	readyzName      = "readyz"
+	exitCode         = 1
+	leaderElectionID = "spanner-autoscaler-leader-election"
```

Comment:
This was created by default with the [hash of repo name](https://github.com/kubernetes-sigs/kubebuilder/blob/72f7c2ef3fd42d21756bc04029ecc499daee2533/pkg/plugins/golang/v2/scaffolds/internal/templates/main.go#L227): `leaderElectionID = "54b82eb3.mercari.com"`. I reverted it to the old one.